moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -175,8 +189,9 @@ const ProjectActivityLog = () => {
                 </TableRow>
               </TableHead>
               <TableBody>
-                {data["moped_activity_log"].map(change => (
+                {reorderEventList(data["moped_activity_log"]).map(change => (
                   <TableRow key={change.activity_id}>
+                    {console.log("Change: ", change)}
```

Comment:
This log might serve an important purpose but wanted to point it out in case it was left behind unintentionally.

Reply:
Ah, thanks for catching this! 🙏 I will have it removed